conan/api/subapi/cache.py

```diff
@@ -199,7 +199,7 @@ def restore(self, path):
                     pkg_layout = cache.create_pkg_layout(pref)  # DB Folder entry
                 # FIXME: This is not taking into account the existence of previous package
                 unzipped_pkg_folder = pref_bundle["package_folder"]
-                out.info(f"Restore: {pref} in {unzipped_pkg_folder}")
+                out.info(f"Restore: {pref} from {unzipped_pkg_folder}")
```

Comment:
It is not restoring from, it is restoring in, the folder is the destination folder

Reply:
Also, unrelated to the PR?